content/lxd/advanced-guide.md

```diff
@@ -473,8 +484,193 @@ and images name with the remote host like:
     lxc exec remoteserver-name:instancename -- apt-get update
    
 
+# Images - Part 2
 
+## Advanced options for Images
 
+1. [Add additional remote (image) servers](#add-remote-servers)
+2. [Manually import an image](#import-images)
+3. [Build your own image](#build-images)
+
+### Import Images
+You can import images, that you:
+
+- built yourself (see [Build Images](#build-images)), 
+- downloaded manually (see [Manual Download](#manual-download))
+- exported from images or containers (see [Export Images](#export-images) and [Create Image from Containers](#create-image-from-containers))
+
+##### Import container image
+
+Components:
+
+- lxd.tar.xz
+- rootfs.squashfs
+
+Use:
+
+	lxc image import lxd.tar.xz rootfs.squashfs --alias custom-imagename
+
+
+##### Import virtual-machine image
+
+Components:
+
+- lxd.tar.xz
+- disk.qcow2
+
+Use:
+
+	lxc image import lxd.tar.xz disk.qcow2 --alias custom-imagename
+
+
+#### Manual download
+You can also download images manually.   
+For that you need to download the components described [above](#import-images).
+
+##### From official LXD imageserver
+
+**Note:** It is easier to use the usual method with `lxc launch`.     
+Use manual download only if you have a specific reason, like modification of the files before use for example.
+
+**Link to official Imageserver:**      
+[https://images.linuxcontainers.org/images/](https://images.linuxcontainers.org/images/)
+
+
+### Export Images
+Use:
+
+	lxc image export imagename [target folder] [flags]
+
+Flags:   
+`--vm` - Query virtual machine images
+
+#### Create Image from Containers
+See command: 
+
+	lxc publish
+
+### Build Images
+For building your own images, you can use [`distrobuilder`](https://github.com/lxc/distrobuilder) (a tool developed by us).
```

Comment:
Feels like this section may be better moved to a getting-started for distrobuilder on the website, then just link to that?

Reply:
:thinking: Well at least some parts are specific to LXD images.
In general the guides in the distrobuilder github repo should be the place for that: 
https://github.com/lxc/distrobuilder/blob/master/doc/index.md
https://github.com/lxc/distrobuilder/blob/master/doc/building.md

But I found that they are not really recent, complete and easy to follow.
Thus I decided to write a short guide, that focuses on LXD images and includes only necessary information and links to the distrobuilder docs for details.